data-prepper-plugins/peer-forwarder/README.md

```diff
@@ -1,138 +0,0 @@
-# Peer Forwarder
```

Comment:
Perhaps we should leave this `README` here with a note that it has been removed. There may be external links to it. But, I'm fine with or without it.